config.yml

```diff
@@ -322,14 +333,14 @@ production_factories:
       - Bake_Bread
       - Bake_Cake
       - Bake_Pumpkin_Pie
-    repair_multiple: 26
+    repair_multiple: 10
     repair_inputs:
       Baked Potato:
         material: BAKED_POTATO
-        amount: 2
+        amount: 5
       Cookie:
         material: COOKIE
-        amount: 4
+        amount: 10
       Bread:
         material: BREAD
```

Comment:
Bread was not doubled here. (should be noted it had not specified an amount, so I assume the factory defaulted to 1 bread previously)